http-servers/html-server.js

```diff
@@ -0,0 +1,50 @@
+const http = require('http'),
+    fs = require('fs'),
+    path = require('path'),
+    whiskers = require('whiskers'),
+    through2 = require('through2');
+
+const port = process.env.port || 8080;
+
+http.createServer((req, res) => {
+    if (req.url.includes('?')) {
```

Comment:
Consider changing this logic to something like a check of a specific query parameter.
E.g. let's enable sendSyncResponse only when "sync=true" query is passed.